packages/core/src/components/EditableText/__tests__/EditableText.test.tsx

```diff
@@ -18,6 +18,17 @@ describe("EditableText", () => {
     expect(input).toBeInTheDocument();
   });
 
+  it("should render a textarea in edit mode with multiline", () => {
+    render(<EditableText multiline value="Editable text" />);
+
+    const component = screen.getByRole("button");
+    fireEvent.click(component);
+
+    const input = screen.queryByRole("input");
+    expect(input).toBeInTheDocument();
+    expect(input.tagName).toBe("TEXTAREA");
```

Comment:
Feels redundant IMO, better just to check that the role is `textbox`  (not `input`)